include/Beman/Optional26/optional.hpp

```diff
@@ -202,22 +203,39 @@ template <class T>
 class optional;
 
 } // namespace beman::optional
+
 namespace std {
+// Since P3168R1: Give std::optional Range Support.
 template <typename T>
 inline constexpr bool ranges::enable_view<beman::optional::optional<T>> = true;
 
+// Since ${PAPER_NUMBER}: ${PAPER_TITLE}.
+// Note: P3168 and P2988 may have different flows inside LEWG/LWG.
+// Implementation of the range support for optional<T&> reflects P3168R1 for now.
+// [optional.iterators], iterator support
 template <typename T>
 inline constexpr bool ranges::enable_view<beman::optional::optional<T&>> = true;
 
+// TODO: document why this is needed.
 template <typename T>
 inline constexpr bool ranges::enable_borrowed_range<beman::optional::optional<T*>> = true;
 
+// TODO: document why this is needed.
 template <typename T>
 inline constexpr bool ranges::enable_borrowed_range<beman::optional::optional<std::reference_wrapper<T>>> = true;
 
+// TODO: document why this is needed.
```

Comment:
This one is correct though.

Reply:
I'll keep it as it is anyways, as it is not related to P3168R1. I removed other overloads as suggested. Thanks!